js/ui/toolbar/ui.toolbar.base.js

```diff
@@ -32,42 +30,44 @@ const DEFAULT_DROPDOWNBUTTON_STYLING_MODE = 'contained';
 
 const TOOLBAR_ITEM_DATA_KEY = 'dxToolbarItemDataKey';
 
-const ToolbarBase = AsyncCollectionWidget.inherit({
-    compactMode: false,
+class ToolbarBase extends AsyncCollectionWidget {
```

Comment:
why did u decide to use `extends` instead of our usual `inherit`?

Reply:
It is a build in and documented functionality to create a child of parent class. In debug mode, we immediately get into the parent function.